provider/consumer.py

```diff
@@ -24,6 +25,8 @@
 from datetime import datetime
 from threading import Thread, Lock
 
+local_dev = os.getenv('LOCAL_DEV', 'False')
```

Comment:
`LOCAL_DEV` is a variable that could be utilized by other parts of the full OpenWhisk deployment (whereas `check_ssl` not so much).  Additionally, `os.getenv()` returns the value as a String, which needs to be transformed into a Boolean anyway. Regardless of what the variable is called, we still need two lines of code to take the environment value and make it a Boolean.